### PR TITLE
docs: Some improvements in the wording of the private testnet guide

### PR DESCRIPTION
**Pull request by msbrogli**
[Rendered](https://github.com/HathorNetwork/rfcs/blob/feat/improve-private-network-guide/text/0033-private-network-guide.md)

### CODE REVIEW

File: text/0033-private-network-guide.md [hunk 1]
```diff
@@ -180,104 +200,108 @@ Include the following block in the file `~/hathor-private-tutorial/conf/privnet.
...
-Now that our fullnodes are in place, the next step is to have a miner to mine blocks in our new private network.
 
-The easiest way to do this in a local testing environment is to run a CPU Miner.
-We have a CPU Miner as a Docker image in https://hub.docker.com/r/hathornetwork/cpuminer
+Now that our full nodes are in place, the next step is to run a miner to find blocks in our new private network.
+
+The easiest way to do this in a local testing environment is to run a CPU miner. You can also use GPU, FPGA, or ASIC miners.
 
-You should run it with the following command, replacing `<address>` by the same address we generated in the beginning of the guide:
+A Docker image of our CPU mining tool is available at https://hub.docker.com/r/hathornetwork/cpuminer
```
[the remark this answers is not in the document]
Comment (luislhl) [reply text/0033-private-network-guide.md:285]: I asked on Slack, and this image is indeed built from the Athos fork

File: text/0033-private-network-guide.md [hunk 2]
```diff
@@ -9,32 +9,52 @@
...
-By the end of the guide you will have a fully working private network with the following setup:
+By the end of the guide, you will have a fully working private network with the following setup:
 
-- Four hathor-core full nodes connected with each other
+- Four hathor-core full nodes connected between each other
 - One cpu-miner mining blocks in the network
 - One tx-mining-service to be used by Wallets to mine transactions
 - Two instances of hathor-wallet-headless configured with different wallets
 
+The diagram below shows the overall architecture of this private network:
```
[the remark this answers is not in the document]
Comment (msbrogli) [reply text/0033-private-network-guide.md:19]: There are two cpu-miners running. One mining only blocks, and a second one mining blocks and transactions.

File: text/0033-private-network-guide.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (msbrogli) [reply text/0033-private-network-guide.md:285]: Great! Thanks for checking.

File: text/0033-private-network-guide.md [hunk 3]
```diff
@@ -289,25 +313,17 @@ docker run -ti --network="host" hathornetwork/cpuminer -t 1 -a sha256d -o stratu
...
-We will run 2 wallet-headles to demonstrate the wallet functionality. They are basically wallets that run as a HTTP api and have most of the same functions as the desktop or mobile wallets.
-
-Each wallet-headless will be connected to one of our fullnodes dedicated to wallets (`fullnode-wallet-1` and `fullnode-wallet-2`). But more than one wallet can use the same fullnode without problem. -->
+## Running the wallets
 
-We will run one wallet-headless to demonstrate the wallet functionality. It's basically a wallet that runs as a HTTP api and have most of the same functions as the desktop or mobile wallets.
+We will run one wallet-headless to demonstrate the wallet functionality. It's similar to any other wallet, but it is fully controlled by an HTTP API.
 
-The first step is to clone the repo https://github.com/HathorNetwork/hathor-wallet-headless
+The first step is to clone the Git repository from https://github.com/HathorNetwork/hathor-wallet-headless
```
[the remark this answers is not in the document]
Comment (msbrogli) [reply text/0033-private-network-guide.md:320]: Great! Thanks!

File: text/0033-private-network-guide.md [hunk 4]
```diff
@@ -180,104 +200,108 @@ Include the following block in the file `~/hathor-private-tutorial/conf/privnet.
...
 - Use `~/hathor-private-tutorial/fullnode-wallet-1` as its data directory in the host.
 - Use the configuration file we create for the network in `hathor.conf.privnet`
-- Enable `wallet index`, which is needed to perform some kinds of operations in the api, like getting transactions information.
+- Enable `wallet index`, which is needed to perform some kinds of operations in the API, like getting transactions information.
 
 
-#### Fullnode Wallet 2
-This fullnode will be used by another of our `hathor-wallet-headless` instances, and we are calling it `fullnode-wallet-2`.
+#### Full node Wallet 2
+This full node will be named `fullnode-wallet-2` because the second wallet will use it. You can safely use one full node for both wallets if you want.
```
[the remark this answers is not in the document]
Comment (luislhl) [reply text/0033-private-network-guide.md:248]: Actually we have just 1 wallet in the guide sending transactions to itself. The initial idea was to have 2, but then I decided to simplify.

**Pull request merged**